config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ OPTIONAL] 
Great job working on this milestone and implementing the requested changes. Please always avoid keeping commented lines of code in your codebase. It makes it not look professional. You can always use git to revisit old codes.

Reply:
@t-yanick Thanks for that! I'll remove it!